src/java.base/share/classes/jdk/crac/Misc.java

```diff
@@ -0,0 +1,41 @@
+package jdk.crac;
+
+import jdk.internal.access.SharedSecrets;
+
+import java.lang.ref.ReferenceQueue;
+
+/**
+ * Additional utilities.
+ */
+public class Misc {
```

Comment:
Given this is a utility class, it should probably be final.  No use in allowing subclasses

Reply:
Right, thanks!